templates/workspace_services/innereye/terraform/firewall.tf

```
@@ -0,0 +1,74 @@
+data "azurerm_firewall" "fw" {
+  name                = "fw-${var.tre_id}"
+  resource_group_name = "rg-${var.tre_id}"
+}
+
+locals {
+  allowedInnerEyeURLs = ["*.anaconda.com", "*.anaconda.org", "binstar-cio-packages-prod.s3.amazonaws.com", "github.com", "*pypi.org", "*pythonhosted.org", "github-cloud.githubusercontent.com"]
```

We can remove github.com and pypi.org from here, since Nexus and Gitea are available.
```suggestion
  allowedInnerEyeURLs = ["*.anaconda.com", "*.anaconda.org", "binstar-cio-packages-prod.s3.amazonaws.com", "*pythonhosted.org", "github-cloud.githubusercontent.com"]
```